src/Makefile.am

```
@@ -519,12 +519,10 @@ libzcash_a_SOURCES = \
   zcash/util.cpp \
   zcash/zip32.cpp
 
-libzcash_a_CPPFLAGS = $(AM_CPPFLAGS) $(PIC_FLAGS) -DBINARY_OUTPUT -DCURVE_ALT_BN128 -DBOOST_SPIRIT_THREADSAFE -fvisibility=hidden -DSTATIC $(BITCOIN_INCLUDES)
+libzcash_a_CPPFLAGS = $(AM_CPPFLAGS) $(PIC_FLAGS) -DBOOST_SPIRIT_THREADSAFE -fvisibility=hidden -DSTATIC $(BITCOIN_INCLUDES)
```

```suggestion
libzcash_a_CPPFLAGS = $(AM_CPPFLAGS) $(PIC_FLAGS) $(BITCOIN_INCLUDES)
```

- We haven't needed `-DBOOST_SPIRIT_THREADSAFE` since #2099 was merged.
- `-fvisibility=hidden` is already handled in `configure.ac` under the `--enable-reduce-exports` config option.
- `-DSTATIC` was for libsnark.